lab_share_lib/processing/rabbit_message.py

```diff
@@ -30,15 +32,15 @@ def subject(self):
     def schema_version(self):
         return self.headers[RABBITMQ_HEADER_KEY_VERSION]
 
-    def decode(self, possible_encoders):
+    def decode(self, possible_encoders: List[AvroEncoderBase]) -> AvroEncoderBase:
         exceptions = []
         for encoder in possible_encoders:
             try:
                 LOGGER.debug(f"Attempting to decode message with encoder class '{type(encoder).__name__}'.")
                 self._decoded_list = list(encoder.decode(self.encoded_body, self.schema_version))
                 if encoder.encoder_type == "binary":
```

Comment:
Can "binary" or alternative values be a constant? it could read similar to following for example, 
`encode.encoder_type == encoder.ENCODER_TYPE_BINARY`